docs/bsp.md

```diff
@@ -212,6 +212,24 @@ An example of use of hierarchical ids is logs, where BSP clients can use the hie
 to improve their readability in the user interface. Clients can show logs in a tree fashion, for
 example, or with dropdowns.
 
+### 1.5.4. DocumentUri
+
+A document URI uniquely identifies a document.
+
+```scala
+/** A id that is a valid URI. https://tools.ietf.org/html/rfc3986 */
```

Comment:
Should we get rid of `DocumentUri` in favor of `Uri`? What is their difference? The spec does not say.

Reply:
It's the same, wanted to keep it to be more similar to LSP, but I agree it's redundant.